src/main/java/uk/gov/hmcts/reform/ccd/document/am/model/CaseDocumentMetadata.java

```diff
@@ -14,11 +16,11 @@
     public CaseDocumentMetadata() {
     }
 
-    public CaseDocumentMetadata(String caseId, String caseTypeId, String jurisdictionId, Document document) {
+    public CaseDocumentMetadata(String caseId, String caseTypeId, String jurisdictionId, List<Document> documents) {
         this.caseId = caseId;
         this.caseTypeId = caseTypeId;
         this.jurisdictionId = jurisdictionId;
-        this.document = document;
+        this.documents = documents;
     }
```

Comment:
Please use the new model entity called CaseDocumentsMetadata with document list.